root-parent-pom/pom.xml

```diff
@@ -326,6 +326,9 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <configuration>
                     <argLine>@{argLine} ${jvm.requiredArgs}</argLine>
+                    <includes>
```

Comment:
see https://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#includes

Reply:
Like the idea very much but please check this won't have surprising interactions with other surefire `includes` in the codebase e.g. Chronicle FIX has profiles that override this. Other projects may also

Reply:
I'm not super keen on it myself. I think the proper fix should be to rename the files to match the convention that every other maven project in the world adheres to. If you create tests using Ctrl-shift-T in IntelliJ they always get a `...Test` suffix. Not sure why we want to support people to deviate from that.

Reply:
@nicktindall If somebody misses naming a test properly, we will literally never know about this neither would we know when the test breaks. It would be nice if it failed for improperly named tests, but skipping those are not acceptable.

Reply:
Sounds like a job for checkstyle https://sevntu-checkstyle.github.io/sevntu.checkstyle/apidocs/com/github/sevntu/checkstyle/checks/coding/NameConventionForJunit4TestClassesCheck.html

Reply:
Apparently Sonar picks it up too, sadly that doesn't break the build https://rules.sonarsource.com/java/tag/tests/RSPEC-3577

Reply:
Well we don't really use Checkstyle or even Sonar universally. If we did that would be a fair point. Right now we have no way to notice it.

@nicktindall @JerryShea I've responded to existing profiles concern in https://github.com/ChronicleEnterprise/RootPom/issues/284 - please discuss

Reply:
I'm of the opinion this change is accommodating sloppiness. I would prefer an approach that stopped the sloppiness at its source by breaking the build (e.g. checkstyle, which could be trivially implemented across our projects by adding a stanza to parent pom, and could address a lot of other issues too). I won't approve it but if someone else approves I won't stand in its way.